ibis/backends/pandas/client.py

```diff
@@ -65,11 +66,20 @@ def from_pandas_extension_dtype(t):
     return getattr(dt, t.__class__.__name__.replace("Dtype", "").lower())
 
 
-@dt.dtype.register(pd.core.arrays.arrow.dtype.ArrowDtype)
-def from_pandas_arrow_extension_dtype(t):
-    import ibis.backends.pyarrow.datatypes as _  # noqa: F401
+try:
+    _arrow_dtype_class = pd.core.arrays.arrow.dtype.ArrowDtype
```

Comment:
This is available at the top level as well `pd.ArrowDtype`

Reply:
Great, thank you!